app/views/images/index.html.haml

```diff
@@ -0,0 +1,2 @@
+No images yet!
+%a(href='images/new') Add an image
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))